### ISSUE TITLE
Publish wagtail-grapple-example image to docker hub to facilitate testing.

### ISSUE THREAD

**Issue by dopry** [GrappleGQL/wagtail-grapple#208]
I'm working on the gatsby 4 update of the wagtail source plugin. It would be nice to have an easy to start wagtail+grapple instance for testing.

**Comment by zerolab** [comment]
Hey @dopry,

I've closed #209 with an explanation to why. Let's move this discussion to https://github.com/GrappleGQL/wagtail-grapple-demo

**Comment by dopry** [comment]
I pushed up an alternate fix that addresses the issues you raise in #209. It should allow you to publish a docker image from this repo in the same release process as the package. It seems like it would save you maintaining and releasing from a second repo in parallel to this one.
